logdevice/ops/ldquery/tables/Directories.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#include "logdevice/ops/ldquery/tables/Directories.h"
+
+#include <queue>
+#include <folly/Conv.h>
+#include <folly/json.h>
+
+#include "../Table.h"
+#include "../Utils.h"
+#include "logdevice/common/configuration/Configuration.h"
+#include "logdevice/common/configuration/ReplicationProperty.h"
+#include "logdevice/common/configuration/UpdateableConfig.h"
+#include "logdevice/common/debug.h"
+#include "logdevice/lib/ClientImpl.h"
+
+
+namespace facebook {
+  namespace logdevice {
+    namespace ldquery {
+      namespace tables {
+
+TableColumns Directories::getColumns() const {
+  return {
+      {"path", DataType::TEXT, "Name of the parent directory."},
+      {"name", DataType::TEXT, "Name of the directory."},
```

Comment:
I think it would be better if we merge those two into a single field `name` instead. Directories are should always be referred to by their absolute path.

Reply:
support both name and parent_dir now. 
But the display for these two columns feels a bit duplicated, we may need better ideas.